ansible/ansible-ai-connect-service
# Remove User Type from markdown user response

Jira Issue: [AAP-27646](https://issues.redhat.com/browse/AAP-27646)


## Description

Remove `User Type` from markdown user response. Currently it is used in the Ansible Lightspeed sidebar view and displayed as either `Licensed` or `Unlicensed`, but it does not provide any practival usage.

## Testing

Tested manually. 

### Steps to test
1. Pull down the PR
2. Start the local server
3. Login to the local server from VS Code and check the sidebar

### Scenarios tested

See above

## Production deployment

     -   There is [an automated testcase in the testing repo](https://github.com/ansible/ansible-wisdom-testing/blob/main/tests/api/test_endpoints.py#L335-L344) and it needs to be updated after this PR is merged.
